compiler/src/Language/Bond/Codegen/Cpp/Types_h.hs

```diff
@@ -241,6 +224,10 @@ namespace std
             fieldInit Field {..} = optional (\x -> [lt|#{fieldName}(#{x})|])
                 $ initValue fieldType fieldDefault
 
+        allocatorType alloc = [lt|using allocator_type = #{alloc};
+
+        |]
```

Comment:
Minor: do we need the empty indentation of just the newlines? If just the newline, we can put the `|]` in the first column.

Reply:
We need indentation for the last line only.